src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -21,12 +22,17 @@
 
 class UserPasswordValidator extends ConstraintValidator
 {
-    private $securityContext;
+    private $tokenStorage;
     private $encoderFactory;
 
-    public function __construct(SecurityContextInterface $securityContext, EncoderFactoryInterface $encoderFactory)
+    /**
+     * @param SecurityContextInterface|TokenStorageInterface
+     *
+     * Passing a SecurityContextInterface as a first argument was deprecated in 2.7 and will be removed in 3.0
+     */
+    public function __construct($tokenStorage, EncoderFactoryInterface $encoderFactory)
```

Comment:
you can actually just typehint `TokenStorageInterface` because `SecurityContextInterface extends TokenStorageInterface`. so people that passed a securitycontext will have no problems.